packages/node_modules/@webex/http-core/src/index.js

```diff
@@ -40,6 +41,10 @@ const protorequest = curry(function protorequest(defaultOptions, options) {
     Reflect.deleteProperty(options, 'json');
   }
 
+  if (!options.timeout) {
+    options.timeout = 60000; // default 60 second timeout
```

Comment:
Let's move this into a `config.js` in the same folder. [Encryption](https://github.com/mattnorris/spark-js-sdk/blob/3ba8c4c442b757808f9dfd4cc4b991013cb260cb/packages/node_modules/%40webex/internal-plugin-encryption/src/config.js#L17) has some good examples.